org.adempiere.base/src/org/compiere/process/ServerProcessCtl.java

```diff
@@ -341,4 +342,21 @@ protected boolean startDBProcess (String ProcedureName)
 		return ProcessUtil.startDatabaseProcedure(m_pi, ProcedureName, m_trx);
 	}   //  startDBProcess
 	
+	/**
+	 * set whether java process call will commit/rollback trx (default is true)
+	 * @param managedTrx
+	 */
+	public void setManagedTrxForJavaProcess(boolean managedTrx)
+	{
+		this.managedTrxForJavaProcess = managedTrx;
+	}
+	
+	/**
+	 * 
+	 * @return true if java process call will commit/rollback trx
+	 */
+	public boolean isManagedTrxForJavaProcess()

```

Comment:
just a question - what is the purpose of this method?  and how to use it when needed?

Reply:
added for unit testing so that trx is not committed upon completion of running a process (for this case, the InOutGenerate process).